### PR TITLE
Testing scenarios

### PR DESCRIPTION
**Pull request by mpangburn**
As demoed at the offsite. Documentation in the corresponding Loop PR.

### CODE REVIEW

**Review by ps2 — APPROVED** [review]
LGTM!

File: MockKit/MockCGMManager.swift [hunk 1]
```diff
@@ -135,6 +135,16 @@ public final class MockCGMManager: TestingCGMManager {
             }
         }
     }
+
+    public func injectGlucoseSamples(_ samples: [NewGlucoseSample]) {
```
Comment (darinkrauss) [inline MockKit/MockCGMManager.swift:139]: Did you want to sort these or doesn't it matter?

Comment (darinkrauss) [reply MockKit/MockCGMManager.swift:139]: Ah, I guess it doesn't matter.

**Review by darinkrauss — APPROVED** [review]
LGTM 🚀

**Pull request merged**